superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -87,33 +85,32 @@ export function fetchFaveStar(sliceId) {
 }
 
 export const SAVE_FAVE_STAR = 'SAVE_FAVE_STAR';
-export function saveFaveStar(sliceId, isStarred) {
-  return function (dispatch) {
+export function saveFaveStar(sliceId: string, isStarred: boolean) {
+  return function (dispatch: Dispatch<ReturnType<typeof addDangerToast>>) {
     const urlSuffix = isStarred ? 'unselect' : 'select';
     SupersetClient.get({
       endpoint: `${FAVESTAR_BASE_URL}/${sliceId}/${urlSuffix}/`,
     })
       .then(() => dispatch(toggleFaveStar(!isStarred)))
-      .catch(() =>
+      .catch(() => {
         dispatch(
           addDangerToast(t('An error occurred while starring this chart')),
-        ),
-      );
+        );
+      });
   };
 }
 
 export const SET_FIELD_VALUE = 'SET_FIELD_VALUE';
-export function setControlValue(controlName, value, validationErrors) {
+export function setControlValue(
+  controlName: string,
+  value: any,
+  validationErrors: any[],
+) {
   return { type: SET_FIELD_VALUE, controlName, value, validationErrors };
 }
 
-export const UPDATE_EXPLORE_ENDPOINTS = 'UPDATE_EXPLORE_ENDPOINTS';
-export function updateExploreEndpoints(jsonUrl, csvUrl, standaloneUrl) {
-  return { type: UPDATE_EXPLORE_ENDPOINTS, jsonUrl, csvUrl, standaloneUrl };
-}
```

Comment:
This action is added in a 4 yo [PR](https://github.com/apache/incubator-superset/pull/1812/files#diff-e6cada89e9d7010ff76d50b8596dba3dc730b59b49436829044e0d698a1a6704R116) but doesn't seem to have been used anywhere (even in the original PR).